applications/mne_scan/mne_scan/mne_scan.pro

```diff
@@ -151,46 +151,48 @@ macx {
     QMAKE_RPATHDIR += @executable_path/../Frameworks
 
     # Copy Resource folder to app bundle
-    filtrc.path = Contents/MacOS/resources/general/
+    filtrc.path = Contents/MacOS/resources/general/default_filters
```

Comment:
Please undo this change. Repeat for all others. Although there are no subfolders in most of these paths we want to stay consistent to when there are subfolders existing.